### ISSUE TITLE
WARNING: Pool 'zfs_pool_scsi0QEMU_QEMU_HARDDISK_disk11' has encountered an uncorrectable I/O failure and has been suspended.

### ISSUE THREAD

**Issue by brianjmurrell** [intel-hpdd/intel-manager-for-lustre#340]
During some test runs, we are seeing the following in the kernel log:

```
[  594.043099] WARNING: Pool 'zfs_pool_scsi0QEMU_QEMU_HARDDISK_disk11' has encountered an uncorrectable I/O failure and has been suspended.
```

This will lead to commands operating on that pool being blocked in the kernel as such:

```
16809 ?        D      0:00  |       \_ umount -t lustre -a
```

**Comment by brianjmurrell** [comment]
Here's [a test run](http://jenkins.lotus.hpdd.lab.intel.com/job/integration-tests-existing-filesystem-configuration/1592/arch=x86_64,distro=el7) with this problem:

```
# ps axf
...
16789 ?        Ss     0:00  \_ sshd: root@notty
16791 ?        Ss     0:00  |   \_ bash -c exec 0<&-; set -e; umount -t lustre -a
16809 ?        D      0:00  |       \_ umount -t lustre -a
...
# dmesg | grep -i zfs
[   43.350561] ZFS: Loaded module v0.7.1-1, ZFS pool version 5000, ZFS filesystem version 5
[  594.043099] WARNING: Pool 'zfs_pool_scsi0QEMU_QEMU_HARDDISK_disk11' has encountered an uncorrectable I/O failure and has been suspended.
```

Happened during:

```
2017-10-11 19:44:15,099 - test - DEBUG - remote_command[lotus-44vm15.lotus.hpdd.lab.intel.com]: umount -t lustre -a
```

**Comment by jgrund** [comment]
This looks to be fixed by #342.
